src/ol/layers/MapboxLayer.js

```diff
@@ -271,9 +273,20 @@ export default class MapboxLayer extends Layer {
   }
 
   updateAttribution(evt) {
-    this.olLayer
-      .getSource()
-      .setAttributions(getMapboxMapCopyrights(evt.target));
+    /**
+     * Since openlayers setAttributes on the source triggers a change event
+     * and may cause the mapbox map to rerender infinitely (e.g. Trafimage-maps),
+     * the copyright is written only when it changes
+     */
+    if (
+      JSON.stringify(this.olLayer.getSource().getAttributions()()) !==
```

Comment:
getAttributions is not reliable,  it's not part of the public api  and  It returns a function in our case.